src/Command/UpdateAdvisoriesDbCommand.php

```diff
@@ -34,7 +34,6 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->checker->update();
-        $output->writeln(sprintf('Database successfully updated'));
```

Comment:
I don't think we need output from this command polluting cron logs 🤔

Reply:
I agree